libfreerdp/core/server.h

```diff
@@ -66,6 +66,8 @@ struct rdp_peer_channel
 	BYTE dvc_open_state;
 	UINT32 dvc_total_length;
 	rdpMcsChannel* mcsChannel;
+
+	char channelName[128];
```

Comment:
Just wondering, if I understand correctly this is a static channel, perhaps we can reduce the array size as the name is limited to 8 or 16 (can't remember) ?

Reply:
no, this is also the dynamic channel struct. did not want to add fancy `malloc` / `free` handling just to have a debug name